driver/gl/raster_cache.go

```diff
@@ -0,0 +1,48 @@
+package gl
+
+import (
+	"image"
+	"os"
+	"sync"
+	"time"
+
+	"fyne.io/fyne"
+)
+
+type rasterInfo struct {
+	pix     *image.RGBA
+	w, h    int
+	alpha   float64
+	expires time.Time
+}
+
+var cacheDuration = (time.Minute * 5)
```

Comment:
Semantics, but cacheDuration should be made `const`.

Reply:
Except that it is settable via ENV vars in init().

Was thinking from the start that it should be settable, but it gets tricky because its driver/gl specific.

Does that work for you ?